### PR TITLE
Update of limit code

### PR DESCRIPTION
**Pull request by hdelanno**
This is an update of the limits code to be closer to what is given for HIG-17-012.
Now the limits produce three kinds of results: ggH, qqH and ppH.
Also the expected values are now blinded and mt plots are automatically produced with pre/post-fit to be able to check pathological cases.
Also the optimize code runs now smoothly at IIHE.
Eventually the README.md has been updated.

This also implements (with some corrections) #402

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,17 +22,18 @@ cd ../..
 scramv1 b -j 16 
 ```
 
-## Submit the limit jobs on lxplus
+## Submit the limit jobs
 example for GGH only (for VBF few updates have still to be done by hand in the code)
 ```bash
-UserCode/llvv_fwk/test/hzz2l2v/computeLimit
+if [[ $HOSTNAME =~ "iihe" ]]; then yes | scp lxplus.cern.ch:/eos/cms/store/user/hbrun/analysis/plotters/plotter_2017_06_08_forLimits_JulyFromAlessio.root $CMSSW_BASE/src/UserCode/llvv_fwk/test/hzz2l2v/computeLimit/.; fi
```
Comment (HuguesBrun) [inline README.md]: Maybe that would be more efficient to have a local copy of the plotter on the IIHE T2 no ?

**Comment by HuguesBrun** [comment]
Hi @hdelanno 
Maybe it would be nice to add the limit plot that you obtain for reference

File: README.md (hunk 1 above)
Comment (hdelanno) [reply README.md]: Why not, a suggestion of where?

**Comment by hdelanno** [comment]
Would you like to see the limit plots just in the PR description or also in a commit?

File: README.md (hunk 1 above)
Comment (HuguesBrun) [reply README.md]: I could copy the plotter in my directory and they you update the compute limit to handle it in case of IIHE ?

**Comment by HuguesBrun** [comment]
I tested this PR on lxplus and it works perfectly!

File: README.md (hunk 1 above)
Comment (HuguesBrun) [reply README.md]: Here: 
`/storage_mnt/storage/user/hbrun/public/plotters/plotter_2017_06_08_forLimits_JulyFromAlessio.root`

**Comment by hdelanno** [comment]
Hi Hugues,
I've implemented your suggestions.
For reference the limit plots are here: http://hdelanno.web.cern.ch/hdelanno/2l2nuPR403/ 
I've not merged in this PR the PR #400 . I let you double check this one :)

**Comment by HuguesBrun** [comment]
perfect ! many thanks !

**Comment by HuguesBrun** [comment]
Hi @hdelanno , i don't know how to re-open this PR, can  you re-open it ?

**Pull request merged**